### PR TITLE
Add comment on international domain names

### PR DESCRIPTION
**Pull request by OR13**
Warn implementers about unicode.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -642,6 +642,32 @@ <h3>
 
       </section>
 
+      <section>
+        <h3>
+          International Domain Names
+        </h3>
+        <p>
+          [[DID-CORE]] identifier syntax does not 
+          allow unicode in method name or method specific identifiers.
```
Comment (TallTed) [inline index.html]:
```suggestion
          allow Unicode in method name nor method specific identifiers.
```

File: index.html [hunk 2]
```diff
@@ -642,6 +642,32 @@ <h3>
...
+        </p>
+        <p>
+          Implementers should be cautious when implementing 
+          support for DID URLs that rely on domain names 
+          or path components that contain unicode characters.
```
Comment (TallTed) [inline index.html]:
```suggestion
          or path components that contain Unicode characters.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Picking nits...

**Review by dmitrizagidulin — APPROVED** [review]
👍 
Thanks @OR13!

**Review by TallTed — APPROVED** [review]

**Pull request merged**